### ISSUE TITLE
Prometheus example for the histogram produces incorrect exposition output

### ISSUE THREAD

**Issue by dc6c** [open-telemetry/opentelemetry-cpp#1432]
**Describe your environment** Windows 10, VS 2019, Bazel 5.0.0

**Steps to reproduce**
1. Build: `bazel build :prometheus_example` from examples/prometheus
2. Run the example: `..\..\bazel-bin\examples\prometheus\prometheus_example.exe histogram`
3. Get the metrics: `curl localhost:8080/metrics`

**What is the expected behavior?**
Output consistent with Prometheus histogram metrics [specification](https://prometheus.io/docs/instrumenting/exposition_formats/#histograms-and-summaries):
1. one entry for `prometheus_metric_example_sum`
2. one entry for `prometheus_metric_example_count`
4. one entry for each value of label "le" of `prometheus_metric_example_bucket` (0, 5, 10, etc. up to "+Inf")

**What is the actual behavior?**
Many entries of each metric, often not with the same value even though the timestamp is the same. E.g.:
```
prometheus_metric_example_sum 2520.7 1654623992100
prometheus_metric_example_sum 1058.2 1654623992100
prometheus_metric_example_sum 0 1654623992100
```
This is just an extract, there were exactly 50 entries when I counted them by piping to `wc -l`. The Prometheus exposition format [specification](https://prometheus.io/docs/instrumenting/exposition_formats/#text-format-details) states "Each line must have a unique combination of a metric name and labels. Otherwise, the ingestion behavior is undefined."

**Comment by lalitb** [comment]
related to #1437. These multiple metric entries are in fact supposed to be different metrics ( with different labels). The Prometheus exporter is not adding labels in the time series data.

**Comment by lalitb** [comment]
fixed by #1440. Please reopen if there are still any issues.
